### ISSUE TITLE
self-hosting: what are the estimated GPU size requirements?

### ISSUE THREAD

**Issue by skeptrunedev** [FlagOpen/FlagEmbedding#18]
Hello, I primarily work on https://github.com/arguflow . We are looking to deploy the Data platform to provide semantic search for a broader range of use-cases. To that end, we need to self-host a production-level embedding model. Is it viable to host bge-large-en ? How big of a GPU will we need? What are the performance expectations? Thanks! 

## Information on our use-case

There is an important dataset in the Argument-Mining research space called DebateSum. Every year, thousands of competitive debate students around the world open source high quality argument mining data that gets buried in hard to parse docx files.

We built a flexible and open source system to parse those files, extract embedding segments, and semantically search them. Further, there is a neat UI to encourage more contributions to the dataset.

Check out the demo @ https://vault.arguflow.ai

Code @ https://github.com/arguflow

**Comment by staoxiao** [comment]
Great! It is viable to host bge-large-en. The memory size used in GPU depends on the batch size and length of your texts. Most of GPUs can load bge models.
